greenriver/hmis-frontend
# 187040557 external forms

## Description

[PT Issue](https://www.pivotaltracker.com/story/show/187040557)

[//]: # 'remove if not applicable'
[Depends on hmis-warehouse PR:
](https://github.com/greenriver/hmis-warehouse/pull/4037)

Note: @martha is the primary author on this PR

## Testing

To test locally:
* see the backend PR for specific testing instructions
* visit https://hmis-warehouse.dev.test/hmis_external_api/external_forms/tchc/prevention_screening
* form submissions should appear in the HMIS 
* give your roles the 'can_manage_external_form_submissions' permission
* add a form rule to enable the prevention form on a given project (or org-wide)
* The submissions table in the interface does not initially include the columns (first name, last name). To make these show up, you may need to set show_in_summary=true on the relevant cdeds


## Type of change
[//]: # 'remove options that are not relevant'


## Checklist before requesting review
